--- origsrc/kdiff3-0.9.96/po/create_qm_files	2011-08-27 05:48:37.000000000 -0500
+++ src/kdiff3-0.9.96/po/create_qm_files	2011-11-02 01:01:56.610765100 -0500
@@ -38,7 +38,7 @@ elif [ "$1" = "install" ]; then
          # (msg2qm is from "qt3-devel-tools"-package.)
          echo Installing /usr/share/locale/$i/LC_MESSAGES/kdiff3.qm
          #sudo /usr/lib/qt3/bin/msg2qm $i/kdiff3.po /usr/share/locale/$i/LC_MESSAGES/kdiff3.qm
-         sudo lconvert -if po -of qm -o /usr/share/locale/$i/LC_MESSAGES/kdiff3.qm $i/kdiff3.po
+         lconvert -if po -of qm -o $DESTDIR/usr/share/locale/$i/LC_MESSAGES/kdiff3.qm $i/kdiff3.po
       fi
    done
 else
